stackslib/src/chainstate/stacks/boot/pox-4.clar

```diff
@@ -912,14 +915,19 @@
           (asserts! (is-eq pox-addr (get pox-addr existing-entry))
                     (err ERR_DELEGATION_WRONG_REWARD_SLOT))
 
+          ;; Validate that amount is less than or equal to `max-amount`
```

Comment:
Commenting on lines 904 to 908 -- this is not actually an issue, because stack increase could only ever be called on something that already passed those checks, but those checks use the wrong let binding:

```clar
          ;; must be stackable
          (try! (minimal-can-stack-stx pox-addr total-ustx reward-cycle u1))
          ;; new total must exceed the stacking minimum
          (asserts! (<= (get-stacking-minimum) total-ustx)
                    (err ERR_STACKING_THRESHOLD_NOT_MET))
```

Should actually be:

```clar
          ;; must be stackable
          (try! (minimal-can-stack-stx pox-addr increased-ustx reward-cycle u1))
          ;; new total must exceed the stacking minimum
          (asserts! (<= (get-stacking-minimum) increased-ustx)
                    (err ERR_STACKING_THRESHOLD_NOT_MET))
```

Can you update that in this PR?

Reply:
Updated accordingly.